package.json

```diff
@@ -22,7 +22,9 @@
     "stylelint": "stylelint 'src/**/*.scss' fix --config stylelint.json",
     "gitSecrets": "git secrets --scan",
     "scanRepo": "git log -p -n 15 | scanrepo",
-    "e2e": "ng e2e",
+    "test:e2e": "protractor e2e-simulator-bdd.conf.js",
+    "test:generate-report": "node generateTestReport.js",
+    "e2e-build": "xcodebuild -workspace ios/App/App.xcworkspace -scheme App -sdk iphonesimulator SYMROOT=$(PWD)/build",
```

Comment:
we probably want to add an additional command in here to make sure it uses the correct config file.

npm run config:dev && xcodebuild -workspace ios/App/App.xcworkspace -scheme App -sdk iphonesimulator SYMROOT=$(PWD)/build"